timesketch/api/v1/resources/sigma.py

```diff
@@ -25,22 +25,52 @@
 
 from sigma.parser import exceptions as sigma_exceptions
 
+from sqlalchemy.exc import IntegrityError
+
 import timesketch.lib.sigma_util as ts_sigma_lib
 
 from timesketch.api.v1 import resources
+
 from timesketch.lib.definitions import HTTP_STATUS_CODE_NOT_FOUND
 from timesketch.lib.definitions import HTTP_STATUS_CODE_BAD_REQUEST
+from timesketch.lib.definitions import HTTP_STATUS_CODE_CONFLICT
+from timesketch.lib.definitions import HTTP_STATUS_CODE_CREATED
+from timesketch.lib.definitions import HTTP_STATUS_CODE_FORBIDDEN
+from timesketch.lib.definitions import HTTP_STATUS_CODE_OK
+
+from timesketch.models.sigma import SigmaRule
+from timesketch.models import db_session
+
 
 logger = logging.getLogger("timesketch.api.sigma")
 
+def _enrich_sigma_rule_object(rule: SigmaRule):
+    """Helper function: Returns an enriched Sigma object givin a SigmaRule.
+
+    Args:
+        SigmaRule: uuid of the rule
+
+    Returns:
+        enriched Sigma dict
+    """
+    parsed_rule = ts_sigma_lib.get_sigma_rule_by_text(rule.rule_yaml)
+    parsed_rule["rule_uuid"] = parsed_rule.get("id",rule.rule_uuid)
```

Comment:
```suggestion
    parsed_rule["rule_uuid"] = parsed_rule.get("id", rule.rule_uuid)
```

Didn't your linter catch this? There are other place in the code where this seems to reoccur.

Reply:
no, but trying to correct it now in this file